manager/integration/tests/test_ha.py

```diff
@@ -2453,8 +2453,9 @@ def test_replica_failure_during_attaching(settings_reset, client, core_api, volu
 
     time.sleep(10)
     wait_for_volume_replica_count(client, volume_name_2, 4)
```

Comment:
@khushboo-rancher @chanow816 do you know what the replica count is 4 at this moment?